controls/V-219195.rb

```diff
@@ -58,5 +58,19 @@
   tag fix_id: 'F-20919r304914_fix'
   tag cci: ['SV-109721', 'V-100617', 'CCI-001494', 'CCI-001495', 'CCI-001493']
   tag nist: ['AU-9', 'AU-9', 'AU-9']
-end
 
+  audit_tools = input('audit_tools')
+
+  if !audit_tools.nil? and !audit_tools.empty?
```

Comment:
please use the `unless` function here rather than the `if !(thing)`
so
```
unless input('audit_tools')nil? or input('audit_tools').empty?`
  ...
else
  ...
end

Reply:
Replaced negative with unless